src/containers/discovery-node-selection-preview/DiscoveryNodeSelectionPreview.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+
+import { Button, ButtonType } from '@audius/stems'
+import { ChangeEvent } from 'react-slick/node_modules/@types/react'
+
+import { usePreviewHotkey } from 'hooks/useHotkey'
+
+import styles from './DiscoveryNodeSelectionPreview.module.css'
+
+const DISCOVERY_NODE_SELECTION_ENABLE_KEY =
```

Comment:
can we just have 1 local storage var called like "enable-dev-mode" or something like that